### PR TITLE
Nueva version CPG-Android 8.0.0

### PR DESCRIPTION
**Pull request by chrisluccas**
# Descripción

## What's Changed
* Added "android:export" by @gussrivero in https://github.com/mercadolibre/fury_cpg-android/pull/162
* Versão 5.2.0 by @chrisluccas in https://github.com/mercadolibre/fury_cpg-android/pull/166
* [Trivial] Bump new Plugin of Gradle by @gussrivero in https://github.com/mercadolibre/fury_cpg-android/pull/163
* Criando a TAG 5.3.0 by @chrisluccas in https://github.com/mercadolibre/fury_cpg-android/pull/167
* Migration to API 32 (Android 12) by @gussrivero in https://github.com/mercadolibre/fury_cpg-android/pull/165
* Release v6.0.0 by @chrisluccas in https://github.com/mercadolibre/fury_cpg-android/pull/168
* Versão 7.0.0 by @chrisluccas in https://github.com/mercadolibre/fury_cpg-android/pull/172
* Trocando url base para chamada a api de header by @chrisluccas in https://github.com/mercadolibre/fury_cpg-android/pull/177
* Release 7.1.0 by @chrisluccas in https://github.com/mercadolibre/fury_cpg-android/pull/178
* Android 13 - Migration by @au-pumigrat in https://github.com/mercadolibre/fury_cpg-android/pull/175

# Ticket ID
- - #....

    Para más información visitar [Wiki.](https://sites.google.com/mercadolibre.com/mobile/arquitectura/allowlist) 

## En qué apps impacta mi dependencia
- [X] Mercado Libre
- [ ] Mercado Pago
- [ ] SmartPOS
- [ ] Alicia: Flex / Logistics
- [ ] WMS
- [ ] Meli Store

### CODE REVIEW

File: android-whitelist.json [hunk 1]
```diff
@@ -693,6 +693,11 @@
       "name": "ui_components",
       "version": "7\\.\\+"
     },
+    {
```
Comment (eduflores72) [inline android-whitelist.json:696]: @chrisluccas te parece agregar un expire para la version de cpg 7.+? y de paso eliminar la version de cpg version 6..+

Comment (eduflores72) [reply android-whitelist.json:696]: @chrisluccas   te parece agregar la misma fecha de expire para ambas versiones ?? asi de esa forma no vamos a olvidarnos de remover ambas porque va ser automatico si agregamos el expire.

Comment (chrisluccas) [reply android-whitelist.json:703]: @eduflores72 Puedes revisarlo de nuevo, por favor ?

Comment (eduflores72) [reply android-whitelist.json:715]: {
      "expires": "2023-05-31",
      "expires": "2023-06-30",
      "group": "com\\.mercadolibre\\.android\\.cpg",
      "name": "ui_components",
      "version": "6\\.\\+|7\\.\\+"
    },
   algo asi tendria que queda, queda mucho mejor

**Review by eduflores72 — APPROVED** [review]

**Review by gussrivero — APPROVED** [review]

**Pull request merged**